Movies-Selection/Movies-Selection/ApiServices/ImageAPIService.swift

```diff
@@ -0,0 +1,36 @@
+// ImageAPIService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import UIKit
+/// Протокол для загрузки изображения
+protocol ImageAPIServiceProtocol {
+    func getImageForMovies(url: URL, complition: @escaping ((UIImage) -> Void))
+    func getImageForDetailsFilm(url: URL, complition: @escaping ((UIImage) -> Void))
+}
+
+/// Сервис получения изображений
+class ImageAPIService: ImageAPIServiceProtocol {
+    // MARK: Internal Methods
+
+    func getImageForMovies(url: URL, complition: @escaping ((UIImage) -> Void)) {
+        DispatchQueue.global().async {
+            guard let imageData = try? Data(contentsOf: url) else { return }
+            DispatchQueue.main.async {
+                guard let image = UIImage(data: imageData) else { return }
+                complition(image)
+            }
+        }
+    }
+
+    func getImageForDetailsFilm(url: URL, complition: @escaping ((UIImage) -> Void)) {
+        DispatchQueue.global().async {
+            guard let detailsDate = try? Data(contentsOf: url)
+            else { return }
+            DispatchQueue.main.async {
+                guard let imageCell = UIImage(data: detailsDate) else { return }
```

Comment:
```suggestion
                guard let image = UIImage(data: detailsDate) else { return }
```
cell сервис не знает для кого фото